client/gov/cli/query.go

```diff
@@ -244,10 +246,10 @@ func GetCmdQueryConfig(storeName string, cdc *wire.Codec) *cobra.Command {
 		Short: "query parameter proposal's config",
```

Comment:
what's the difference between this cmd and the 'GetCmdQueryGovConfig' ? who call the 'GetCmdQueryConfig' ?

Reply:
'GetCmdQueryConfig'  is the  previous method. I could delete it.